receiver/awsxrayreceiver/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package awsxrayreceiver implements a receiver that can be used by the
+// Opentelemetry collector to receive traces in the AWS X-Ray segment format.
+// More details can be found on:
+// https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html
+package awsxrayreceiver
```

Comment:
add a new line before this

Reply:
lint failed. So should not have a newline. Resolved